docs/user-guide/continuous-aggregation/usecase-example.md

```diff
@@ -0,0 +1,88 @@
+# Usecase Example
+Following are three major usecase examples for continuous aggregation:
+
+1. **Real-time Analytics**: A real-time analytics platform that continuously aggregates data from a stream of events, delivering immediate insights while optionally downsampling the data to a lower resolution. For instance, this system can compile data from a high-frequency stream of log events (e.g., occurring every millisecond) to provide up-to-the-minute insights such as the number of requests per minute, average response times, and error rates per minute.
+
+2. **Real-time Monitoring**: A real-time monitoring system that continuously aggregates data from a stream of events and provides real-time alerts based on the aggregated data. For example, a system that aggregates data from a stream of sensor events and provides real-time alerts when the temperature exceeds a certain threshold.
+
+3. **Real-time Dashboard**: A real-time dashboard that shows the number of requests per minute, the average response time, and the number of errors per minute. This dashboard can be used to monitor the health of the system and to detect any anomalies in the system.
+
+In all these usecases, the continuous aggregation system continuously aggregates data from a stream of events and provides real-time insights and alerts based on the aggregated data. The system can also downsample the data to a lower resolution to reduce the amount of data stored and processed. This allows the system to provide real-time insights and alerts while keeping the data storage and processing costs low.
+
+## Real-time Analytics Example
+
+Consider a usecase where you have a stream of log events from a web server that you want to analyze in real-time. The log events contain information such as the status of the request, the size of the response, the client IP address, and the timestamp of the request. You want to continuously aggregate this data to provide real-time analytics on the number of requests per minute, the min/max/average packet size, and the error rate per minute. Then the query for continuous aggregation would be:
+
+```sql
+CREATE FLOW ngx_aggregation
+SINK TO ngx_statistics
+AS 
+SELECT 
+    status,
+    count(client) AS total_logs,
+    sum(case when status >= 400 then 1 end) as error_logs,
+    min(size) as min_size,
+    max(size) as max_size,
+    avg(size) as avg_size
+FROM ngx_access_log 
+GROUP BY
+    status,
+    date_bin(INTERVAL '1 minutes', access_time, '2024-01-01 00:00:00'::Timestamp);
+```
```

Comment:
Please do not repeat the same example, consider to remove these content and add some description and a link to the example in overview.md